docs/UserGuide.md

```diff
@@ -129,18 +129,18 @@ Amigos provides functionality to manage logs, which are essentially detailed not
 
 ### Adding a log: `addlog`
 
-Adds a log to an existing friend in Amigos.
+Adds a log to an existing friend at the specified `INDEX` in Amigos.
+The `INDEX` refers to the index number shown in the displayed person's list.
+Format: `addlog INDEX t/[TITLE] d/[DESCRIPTION]`
 
-Format: `addlog n/[NAME] t/[TITLE] d/[DESCRIPTION]`
-
-* The `NAME` field is compulsory.
+* The `INDEX` field is compulsory.
 * If the `TITLE` argument is provided, then the `DESCRIPTON` argument is optional.
 * If neither `TITLE` nor `DESCRIPTION` arguments are provided, then a GUI
   pop up will prompt the user to key in the title and longer-form text as the description.
 
 Examples:
-* `addlog n/John doe t/has a pet named poki`
-* `addlog n/Andrew Tan t/recommended movies d/the martian, interstellar, three idiots`
+* `addlog 1 t/has a pet named poki`
+* `addlog 2 t/recommended movies d/the martian, interstellar, three idiots`
```

Comment:
Perhaps adding logs based on a friend's name may be more aligned to how Amigos work in general, where most commands identify a friend based on his/her name? For deletefriend n/name, I iterated through the unfiltered list from `model.getAddressBook().getPersonList()` and made use of the `isSamePerson` method to identify and extract a friend with the same name. This will allow users to add logs to a friend not on the displayed person list too!

Reply:
Main reason is that it circumvents the need for searching for a friend first. Simplifies the command significantly. We could change to name-based down the line, but as an MVP I think index is viable.

Reply:
@dionegoh makes sense, can I suggest that for log-related features, name-based commands be implemented in `v1.2b` and index-based ones be implemented for friend-related features too? Numbers are after all faster than typing names! 

i.e. we push these changes to `v1.2b` or even `v1.3` as optimising for CLI use.